packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -371,7 +361,8 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
     pageTo(((_rowCount - 1) / widget.rowsPerPage).floor() * widget.rowsPerPage);
   }
 
-  bool _isNextPageUnavailable() => !_rowCountApproximate &&
```

Comment:
this should be a regular function instead, the flutter style guide suggest s arrow function should be one liner.